laws/src/main/scala/algebra/laws/Rules.scala

```diff
@@ -3,11 +3,17 @@ package laws
 
 import org.scalacheck.{Arbitrary, Prop}
 import org.scalacheck.Prop._
+import Prop.{False, Proof, Result}
 
 import algebra.std.boolean._
 
 object Rules {
 
+  def serializable[M](m: M): (String, Prop) =
+    "serializable" -> Prop { _ =>
+      Result(status = if (m.isInstanceOf[Serializable]) Proof else False)
```

Comment:
This really can't fail now since we are extending serializable in the base case. A stronger test would be to serialize and deserialize and not see an exception. That's a stronger constraint, but without doing that the serializable marker is not very useful.